### ISSUE TITLE
Error: Could not determine serverBuildDirectory in Remix config in postbuild step

### ISSUE THREAD

**Issue by smerchek** [metronome-sh/metronome-sh#2]
👋 I had trouble during the postbuild. Here's the error details:

```

> web@1.0.0 build:remix
> remix build && metronome postbuild

Building Remix app in production mode...
Built in 420ms
metronome postbuild

Uploads the project build metadata to Metronome

Options:
  --version  Show version number                                       [boolean]
  --help     Show help                                                 [boolean]

Error: Could not determine serverBuildDirectory in Remix config
    at getMetadata (/Users/scott/code/udisc/web/node_modules/metronome-sh/dist/cli/index.js:87:15)
    at uploadMetadata (/Users/scott/code/udisc/web/node_modules/metronome-sh/dist/cli/index.js:104:28)
    at Object.postbuild [as handler] (/Users/scott/code/udisc/web/node_modules/metronome-sh/dist/cli/index.js:126:11)
    at /Users/scott/code/udisc/web/node_modules/yargs/build/index.cjs:1:9054
    at j (/Users/scott/code/udisc/web/node_modules/yargs/build/index.cjs:1:4931)
    at M.applyMiddlewareAndGetResult (/Users/scott/code/udisc/web/node_modules/yargs/build/index.cjs:1:9023)
    at M.runCommand (/Users/scott/code/udisc/web/node_modules/yargs/build/index.cjs:1:7206)
    at Jt.[runYargsParserAndExecuteCommands] (/Users/scott/code/udisc/web/node_modules/yargs/build/index.cjs:1:56715)
    at Jt.parse (/Users/scott/code/udisc/web/node_modules/yargs/build/index.cjs:1:38938)
    at Jt.get [as argv] (/Users/scott/code/udisc/web/node_modules/yargs/build/index.cjs:1:60204)
ERROR: "build:remix" exited with 1.
```

```
❯ npx metronome --version
1.2.0
```

Remix version: 1.3.3

remix.config.js

```
/**
 * @type {import('@remix-run/dev/config').AppConfig}
 */
module.exports = {
  cacheDirectory: './node_modules/.cache/remix',
  ignoredRouteFiles: ['.*', '**/*.css', '**/*.test.{js,jsx,ts,tsx}'],
};
```

I'm wondering if this is because https://remix.run/docs/en/v1/api/conventions#serverbuilddirectory is deprecated now? I based this config off of the new [blues-stack](https://github.com/remix-run/blues-stack).

**Comment by ErickTamayo** [comment]
Thanks for opening this! I released v1.3.0 which should solve this issue. Please update `metronome-sh`

**Comment by smerchek** [comment]
Yes, that fixed it. Thanks!
